src/test/java/org/folio/search/service/ResourceIdServiceTest.java

```diff
@@ -167,7 +167,7 @@ void streamResourceIds_positive_emptyCollectionProvided() throws IOException {
   @Test
   void streamResourceIdsInTextTextType_positive_emptyCollectionProvided() {
     mockSearchRepositoryCall(emptyList());
-    when(queryConverter.convertForConsortia(TEST_QUERY, RESOURCE_NAME)).thenReturn(searchSource());
+    when(queryConverter.convertForConsortia(TEST_QUERY, RESOURCE_NAME, TENANT_ID)).thenReturn(searchSource());
     when(properties.getScrollQuerySize()).thenReturn(QUERY_SIZE);
```

Comment:
You changed existing tests to use the new method. However, an old method still exists. Please revert this test changes and right new tests for new method.

Reply:
I've changed this method inside ResourceIdServiceTest.java and changed corresponding tests. 
Yes, the old method still exists, but inside this class it's never invoked.